### PR TITLE
Version 1.0.0

### PR DESCRIPTION
**Pull request by hirasso**
**Description**

Release version 1.0.0 to signal production readiness.

**Drive-By**

- Update and clean-up dependencies
- Minor wording fixes in the readme

**Checks**

- [x] The PR is submitted to the `master` branch
- [x] The code was linted before pushing (`npm run lint`)
- [x] All tests are passing (`npm run test`)
- [ ] ~New or updated tests are included~
- [x] The documentation was updated as required

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,6 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.39.0",
-    "@swup/cli": "^5.0.1",
```
Comment (daun) [inline package.json:58]: Maybe I'm missing something -- isn't this required for the build step?

**Review by daun — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (hirasso) [reply package.json:58]: Ooops, right!! Wait

Comment (hirasso) [reply package.json:58]: ... I got confused because parallel plugin was not yet using @swup/cli. Re-installed it.

**Comment by daun** [comment]
🎊

**Pull request merged**